src/main/java/org/cactoos/text/CapitalizedText.java

```diff
@@ -53,15 +53,15 @@ public CapitalizedText(final Text text) {
     @Override
     public String asString() throws IOException {
         return new JoinedText(
-            new TransformedIterable<String, Text>(
+            new TransformedIterable<Text, Text>(
                 new SplitText(this.origin),
-                new Func<String, Text>() {
+                new Func<Text, Text>() {
                     @Override
-                    public Text apply(final String input) throws Exception {
+                    public Text apply(final Text input) throws Exception {
```

Comment:
@Timmeey I agree totally with you, but I'm totally against private classes or methods. What do you think I create a new class called `CapitalizeTextFunc` and instantiate an object instead of this func?